index.html

```diff
@@ -59,11 +58,149 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">
+                            <h3 class="ptitle">Multi-Post Stories</h3>
+                        </div>
+                        <uL class="details">                           
+                                
+                            <li class="conpoy"> CANOPY</li>                             
+                            <li class="backeddev">Back End Dev</li>                               
+                            <li class="text2015">2015</li> 
+                        </uL>  
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext2">About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+
+
+                <h4 class="letconnect"> LET’S CONNECT</h4>
+                           
+                       
+
+                    
+                <div class="socialmedia">
+                    <ul class="media">                           
+                        
+                        <li><a class="media"><img src="img/media-3.png"></a></li>
+                        <li><a class="media"><img src="img/media-4.png"></a></li>
+                        <li><a class="media"><img src="img/media-1.png"></a></li>
+                        <li><a class="media"><img src="img/media-5.png"></a></li>
+                        <li><a class="media"><img src="img/media-2.png"></a></li>
+                    </ul>
+
+                </div>
+
+                <a class="button">Get my resume</a>
+
+                <div class="skills">
+                    <h2>Languages</h2>
+                    <ul class="skilssitem">                       
+                        <li> <img src="img/Langguage.png"></li>
+                        <li>Frameworks</li>
+                        <li>Skills</li>
+                    </ul>
+                </div>
+                
+            </section> 
 
        
 
```

Comment:
- [ ] @shahadat3669 has done great help by giving you detailed feedback on what and how to improve certain sections but it seems you have not been able to capture all that. I am suggesting strongly that you forget about the deadline for a bit - like you indicated and focus on making this work. Clients do have specifications and well as deadlines. This is the culture we all are getting used to in Microverse, thanks to @microverse. Kindly happy all changes suggested by the previous reviewer in [this thread/conversation](https://github.com/samuelkinyanjui/Portfolio-mobile-first/pull/3#pullrequestreview-1459155628) about spacing, appropriate font properties, structural positioning among others, just to make sure that you replicate the Figma design as close as possible.
- I think that avoiding the use of absolute units on widths and heights will help improve your page's look - `%`, `vh`, `vw`.